client/my-sites/checkout/checkout-thank-you/header.jsx

```diff
@@ -56,28 +47,7 @@ export class CheckoutThankYouHeader extends PureComponent {
 			return translate( 'You will receive an email confirmation shortly.' );
 		}
 
-		if ( isDelayedDomainTransfer( primaryPurchase ) ) {
-			return translate(
-				'Your new site is now live, with a temporary domain. Start the transfer to get your domain ' +
-					'{{strong}}%(domainName)s{{/strong}} moved to WordPress.com.',
-				{
-					args: { domainName: primaryPurchase.meta },
-					components: { strong: <strong /> },
-				}
-			);
-		}
-
-		return translate(
-			"You will receive an email confirmation shortly for your purchase of {{strong}}%(productName)s{{/strong}}. What's next?",
-			{
-				args: {
-					productName: primaryPurchase.productName,
-				},
-				components: {
-					strong: <strong />,
-				},
-			}
-		);
```

Comment:
Curiosity, not feedback, to check my own "math":

Was this removal related to the failure page changes? Looking at the code it seems like it just wasn't being used as this header just wasn't being called for delayed domain transfer purchases, or even ecommerce purchases, before that bit of logic was [removed](https://github.com/Automattic/wp-calypso/pull/88572/files#diff-daac408bb350333e9fb8ca7c03233e75f03656e02a45c152e574227e6d253ee7L652-L675).

Again, not questioning the PR, just curious to confirm if I'm reading the history right 🙂

Reply:
Your "math" is completely accurate and correct 🙂 👍